lingua_franca/parse.py

```diff
@@ -200,9 +201,11 @@ def extract_duration(text, lang=None):
         return extract_duration_en(text)
     if lang_code == "cs":
         return extract_duration_cs(text)
+    if lang_code == "de":
+        return extract_duration_cs(text)
 
     # TODO: extract_duration for other languages
-    _log_unsupported_language(lang_code, ['en','cs'])
+    _log_unsupported_language(lang_code, ['en','cs', 'de'])
```

Comment:
This entry should've been there before, but you're adding support for `de`, so unless I need more coffee I don't think we need it anymore.

Reply:
Get the coffee ready, since the master's parse.py

> 
>    if lang_code == "en":
>         return extract_duration_en(text)
>     if lang_code == "cs":
>         return extract_duration_cs(text)
> 
>     # TODO: extract_duration for other languages
>     _log_unsupported_language(lang_code, ['en','cs'])

Reply:
Ah. Mea culpa, I changed that bit's behavior in a refactor I've been working on. I forgot we were passing the supported list explicitly.